owncloud/test/test.py

```diff
@@ -567,9 +567,15 @@ def test_copy_to_non_existing_dir(self):
         )
 
     @data_provider(files)
-    def test_share_with_link(self, file_name):
+    def test_share_with_link_lesser_8_2(self, file_name):
         """Test sharing a file with link"""
 
+        current_version = self.client.get_version().split('.')
+        current_version_major = current_version[0]
+        current_version_minor = current_version[1]
+        if ((current_version_major >= '8') and (current_version_minor >= '2')):
+            raise unittest.SkipTest("This test should not run against >=8.2 servers");
```

Comment:
No, that string is correct, which is not correct is the logic, it should check the major before, I'll change it.